WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -11,6 +11,15 @@ struct PointOfSaleDashboardView: View {
         self.totalsViewModel = viewModel.totalsViewModel
     }
 
+    var shouldBeHidden: Bool {
```

Comment:
nitpick;

There can be other internal views that will need to be hidden when other conditions are met.

```suggestion
    private var isEmptyOrError: Bool {
```

Or 

```suggestion
    private var isCartShown: Bool {
```

Reply:
Consistency-wise, should we consider moving such a check from a `View` to `ViewModel`, as it's done in `ItemListViewModel`?

Reply:
Yeah, I agree. Moved on https://github.com/woocommerce/woocommerce-ios/pull/13361/commits/63309c7634cadc32f22eb88308c34072430e8fc3 I used `isCartShown` so we're specific as we're using within the `.renderedIf()` property 👍